app/src/main/java/org/simple/clinic/home/overdue/OverdueAppointmentRow.kt

```diff
@@ -131,10 +136,23 @@ data class OverdueAppointmentRow(
         "$overdueDays"
     )
 
+    holder.diagnosisTextView.text = diagnosisText()
+
     updateBottomLayoutVisibility(holder)
     updatePhoneNumberViewVisibility(holder)
   }
 
+  private fun diagnosisText(): CharSequence {
+    return listOf(
+        diagnosedWithDiabetes to "Diabetes",
+        diagnosedWithHypertension to "Hypertension"
```

Comment:
These need to be translated. Please add them in `strings.xml`

Reply:
Done